lib/Bio/EnsEMBL/DataCheck/Checks/GeneBiotypes.pm

```diff
@@ -136,9 +136,18 @@ sub biotype_groups {
     }
   }
 
-  my $desc_1 = 'genes have at least one transcript with a matching biotype group';
-  is(scalar(@group_mismatch), 0, $desc_1);
-
+  my $mca = $self->dba->get_adaptor('MetaContainer');
+  if($mca->single_value_by_key('genebuild.method') eq 'projection_build'){
+      if (scalar(@group_mismatch) > 100){
+	  my $desc_proj = 'genes have at least one transcript with a matching biotype group';
```

Comment:
I would change the message to something more meaningful there:
```suggestion
	  my $desc_proj = 'projected genes have at most 100 transcripts with a matching biotype group';
```
may be?

Reply:
updated to 'found no more that 100 genes with mismatched transcript biotype groups'